src/main/scala/com/azavea/modellab/Service.scala

```diff
@@ -134,7 +177,8 @@ object Service extends SimpleRoutingApp with DataHubCatalog with Instrumented wi
       } ~
       pingPong ~
       pathPrefix("layers"){registerLayerRoute} ~
-      pathPrefix("breaks"){registerColorBreaksRoute}
+      pathPrefix("intbreaks"){registerIntColorBreaksRoute} ~
+      pathPrefix("floatbreaks"){registerFloatColorBreaksRoute}
```

Comment:
This feels strange, but I'm not sure the best option
@caseypt could you please select from the menu:
1. `/intbreaks` and `/floatbreaks` Ship it!
2. `/breaks?type=float` and `/breaks?type=int`, with `int` being default to not break the API
3. `/breaks` with `POST`

```json
{
 "type": "float",
 "breaks": "..."
}
```